School.js

```diff
@@ -19,6 +104,7 @@ class School {
   addClass(name, teacher) {
     let newClass = new Class(name, teacher);
     this.classes[name] = newClass;
+
```

Comment:
You might want to `return newClass` here, so you could use it later in the app.js file to include in your res.json response object.